src/DataStructures/DataBox/DataBox.hpp

```diff
@@ -1708,7 +1708,7 @@ SPECTRE_ALWAYS_INLINE constexpr auto mutate_apply(
  * DataBox, `box`
  */
 template <typename F, typename BoxTags, typename... Args>
-SPECTRE_ALWAYS_INLINE constexpr auto mutate_apply(
+SPECTRE_ALWAYS_INLINE constexpr void mutate_apply(
```

Comment:
The other `mutate_apply` overloads can return, is this overload perhaps just missing the `return`?